src/remotes/member.d.ts

```diff
@@ -3,3 +3,5 @@ interface MemberResponseInterface {
   nickName: string;
   email: string;
 }
+
+type UserInformationInterface = Omit<MemberResponseInterface, 'id'>;
```

Comment:
```suggestion
interface UserInformationInterface = Omit<MemberResponseInterface, 'id'>;
// type UserInformationType = Omit<MemberResponseInterface, 'id'>;
```

명칭과 타입을 통일하면 좋을 거 가타요!!